php8.0.Dockerfile

```diff
@@ -13,13 +13,15 @@ RUN composer update --ignore-platform-reqs --optimize-autoloader \
     
 FROM php:8.0-cli-alpine as final
 
+ENV PHP_MEMCACHED_VERSION=v3.2.0
```

Comment:
Checked with @christyjacob4 and he's okay with using this library and this version